backend/api/prisonApi.js

```diff
@@ -248,6 +248,9 @@ const prisonApiFactory = client => {
   const getCaseNoteSummaryByTypes = (context, params) =>
     get(context, `/api/case-notes/summary?${mapToQueryString(params)}`)
 
+  const getCaseNote = (context, offenderId, caseNoteId) =>
+    get(context, `/api/offenders/${offenderId}/case-notes/${caseNoteId}`)
```

Comment:
This needs to go into the caseNotesApi.js module

Reply:
Check to see if there is already an existing method in the caseNotesApi.js module.

Reply:
not used will remove

Reply:
not used will remove